src/common/hashtable/src/stack_hashtable.rs

```diff
@@ -193,13 +188,8 @@ where
                 _alignment: [0; 0],
             }));
         }
-        while (self.table.len() + other.table.len()) * 2 > self.table.capacity() {
```

Comment:
No need `while` to loop?

Reply:
It's duplicated. The inner method will check it.

Reply:
The inner Table0 will check it. So it's duplicated here.